index.html

```diff
@@ -88,6 +88,10 @@ <h3>Options</h3>
       <input id=override-native type="checkbox" checked>
       Override Native (reloads player)
     </label>
+    <label>
+      <input id=mirror-source type="checkbox" checked>
+      Mirror sources from player.src (reloads player)
```

Comment:
Should we just call this `sourceset`? This the target of these pages are us, we can be as technical as we want.

Reply:
At first I wanted to, but that isn't really the feature that we care about here. We care about mirroring sources from `player.src`, `sourceset` just happens to make that very easy to do.

Reply:
Maybe adding `uses sourceset` in the parens too.
My thought process here is that we always want mirroring but that it's possible that sourceset may cause issues so we'd want to disable it.

Reply:
I will do that and merge